prayer_times_calculator/pray_times_calculator.py

```diff
@@ -38,6 +39,19 @@ def __init__(
         school="",
         midnightMode="",
         latitudeAdjustmentMethod="",
+        tune = bool,
+        imsak_tune = 0,
+        fajr_tune = 0,
+        sunrise_tune = 0,
+        dhuhr_tune = 0,
+        asr_tune = 0,
+        maghrib_tune = 0,
+        sunset_tune = 0,
+        isha_tune = 0,
+        midnight_tune = 0,
+        fajr_angle = "15",
```

Comment:
should we set the defaults to 15 here?

Reply:
actually that is not necessary to set the default to 15 here, i set it to 15 there just to make it clear that aladhan api will use 15 as default even when we not set it.

Reply:
or maybe if you prefer to leave it to ommit the default  value, i will change it.

Reply:
done.